### DISCUSSION TITLE
SD Transfer Utility Requirements

### DISCUSSION THREAD

**Discussion by tychoc** [BlueSCSI/BlueSCSI-v2#95] [General]
I’ve noticed the SD Transfer utility only works if the BlueSCSI is setup to present a drive to the Mac. Is this expected behavior? I have one setup just as a network device, no drives, and the Transfer utility says it can’t find a BlueSCSI. If I add a drive for the BlueSCSI to present to the Mac, the transfer utility works.

**Comment by erichelgeson** [comment]
Yes currently one fixed disk is required to send commands back and forth. I could loosen it up to allow any type (and maybe that makes more sense now that we have a network adapter type).
